python/lsst/daf/persistence/posixStorage.py

```diff
@@ -610,6 +612,70 @@ def writeFitsStorage(butlerLocation, obj):
             persistence.persist(obj, storageList, butlerLocation.getAdditionalData())
 
 
+def readParquetStorage(butlerLocation):
+    """Read from a butlerLocation.
+
+    The object returned by this is expected to be a subtype
+    of `ParquetTable`, which is a thin wrapper to `pyarrow.ParquetFile`
+    that allows for lazy loading of the data.  This is notably 
+    a different object from what gets passed to the butler `put` function, 
+    which should be a `pandas.DataFrame`.
+
+    Parameters
+    ----------
+    butlerLocation : ButlerLocation
+        The location & formatting for the object(s) to be read.
+
+    Returns
+    -------
+    A list of objects as described by the butler location. One item for
```

Comment:
The original purpose of the list of locations was to allow a single dataset to be assembled from multiple physical serializations.  If you're not actually expecting/using the list, I would just use the initial entry (and perhaps even warn if there is more than one).